src/shared/modules/visualization/visualizationDuck.js

```diff
@@ -38,11 +38,13 @@ function updateLabelData (state, labels) {
 }
 
 function updateStyleData (state, styleData) {
-  return Object.assign({}, state, {styleData: styleData})
+  const clonedStyle = Object.assign({}, styleData)
```

Comment:
Oskar: This is what I was trying to talk about at standup. Do I need to do a deep copy to keep the redux state immutable immutable? This works for now in practice but you wont be able to do fancy stuff like rewind/replay.